index.js

```diff
@@ -129,48 +130,35 @@ internals.parse = function (xml, req, cb) {
 };
 
 
-internals.query = function (req, apiDomain, options) {
-
-  req.SignatureMethod = 'HmacSHA256';
-  req.SignatureVersion = 2;
-  req.AWSAccessKeyId = options.key;
-  req.Timestamp = new Date().toISOString();
-
-  // Sign...
-  // Request keys must be sorted with natural byte ordering.
-  // http://docs.aws.amazon.com/AlexaWebInfoService/latest/index.html?CalculatingSignatures.html
-  const keys = Object.keys(req).sort();
-  const q = keys.reduce((memo, k) => {
-
-    if (memo) {
-      memo += '&';
-    }
-    // Manually replace single quotes with `%27` as `encodeURIComponent` doesnt
-    // seem to encode them, and things break:
-    // https://github.com/wrangr/awis/issues/3
-    const value = encodeURIComponent(req[k]).replace(/'/g, '%27');
-    return memo + encodeURIComponent(k) + '=' + value;
-  }, '');
-  const tmpl = 'GET\n%s\n/\n%s';
-  const stringToSign = Util.format(tmpl, apiDomain, q);
-  const signature = Crypto.createHmac('SHA256', options.secret);
-  signature.update(stringToSign);
-  req.Signature = signature.digest('base64');
-
-  return req;
-};
-
-
 module.exports = function (options) {
 
   return function (req, cb) {
 
-    const apiDomain = (req.Action === 'TopSites') ? 'ats.amazonaws.com' : 'awis.amazonaws.com';
+    const host = (req.Action === 'TopSites') ? 'ats.us-west-1.amazonaws.com' : 'awis.us-west-1.amazonaws.com';
```

Comment:
@Janpot Is the `us-west-1` region hard-coded intentionally? Are there any adverse effects to doing this?

Reply:
The region seems to be required. I tried without it and it would complain about it:

`Credential should be scoped to a valid region, not 'us-east-1'.`

I have made this configurable via the `options` and documented it in the [`README.md`](https://github.com/wrangr/awis#usage)